strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generation/License.java

```diff
@@ -0,0 +1,35 @@
+package org.carlspring.strongbox.artifact.generation;
+
+public class License
+{
+    private String type;
+    private String url;
+    
+    public License()
+    {
+
+    }
+    
+    public License(String type, String url)
+    {
+        this.setType(type);
+        this.setUrl(url);
+    }
+
+	public String getType() {
+		return type;
+	}
+
+	public void setType(String type) {
+		this.type = type;
+	}
+
+	public String getUrl() {
+		return url;
+	}
+
+	public void setUrl(String url) {
+		this.url = url;
+	}
+    
```

Comment:
Cosmetic:
* Braces on new line, please.
* We use spaces instead of tabs, as you can see in the code above yours. One "tab" is 4 spaces.